### PR TITLE
Community-Stable OracleLinux: Add Dependabot capability

### PR DESCRIPTION
**Pull request by kiazhi**
## PR Summary

Not sure if I'm getting this correct.

- Added dependabot Dockerfile into community-stable OracleLinux folder
- Updated Dependabot yaml file with the configuration
- Updated the meta file of OracleLinux

## PR Checklist

- [X] [PR has a meaningful title](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
  - Use the present tense and imperative mood when describing your changes
- [X] [Summarized changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [X] [Change is not breaking](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#making-breaking-changes)
- [X] [Make sure all `Dockerfile`, `.sh`, `.ps1` and `.psm1` files have the correct copyright header](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [X] This PR is ready to merge and is not [Work in Progress](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---work-in-progress).
  - If the PR is work in progress, please add the prefix `WIP:` to the beginning of the title and remove the prefix when the PR is ready.
- **Adds a new image or tag**
  - [ ] Not Applicable
- **OR**
  - [ ] Update [README.powershellcommunity.md](https://github.com/PowerShell/PowerShell-Docker/blob/master/docs/README.powershellcommunity.md)
  - [ ] Update [vsts-ci.yml](https://github.com/PowerShell/PowerShell-Docker/blob/master/vsts-ci.yml)

### CODE REVIEW

**Review by RDIL — APPROVED** [review]

File: release/community-stable/oraclelinux/meta.json [hunk 1]
```diff
@@ -3,7 +3,7 @@
     "PackageFormat": "powershell-${PS_VERSION}-1.rhel.7.x86_64.rpm",
     "SkipGssNtlmSspTests": false,
     "shortTags": [
-        {"Tag": "latest"}
+        {"Tag": "8.2"}
     ],
     "tagTemplates": [
         "#psversion#-oraclelinux-#tag#",
```
Comment (anamnavi) [inline release/community-stable/oraclelinux/meta.json:9]:
```suggestion
        "#psversion#-oraclelinux-#shorttag#",
```

Comment (anamnavi) [reply release/community-stable/oraclelinux/meta.json:9]: I'm not 100% sure of this though as I'm new on the project, so asking @TravisEz13 if this is indeed a change that might be wanted for Linux images

Comment (RDIL) [reply release/community-stable/oraclelinux/meta.json:9]: Makes sense to me.

File: .github/dependabot.yml [hunk 2]
```diff
@@ -42,3 +42,8 @@ updates:
     directory: "/release/preview/windowsservercore/dependabot"
     schedule:
       interval: "weekly"
+
+  - package-ecosystem: "docker"
+    directory: "/release/community-stable/oraclelinux/dependabot"
```
Comment (TravisEz13) [inline .github/dependabot.yml:47]:
```suggestion
    directory: "/release/community-stable/oraclelinux/docker"
```

File: release/community-stable/oraclelinux/dependabot/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,6 @@
+# Copyright (c) Microsoft Corporation. 
+# Licensed under the MIT License.
```
Comment (TravisEz13) [inline release/community-stable/oraclelinux/dependabot/Dockerfile:2]: delete this file, this was only needed when we dynamically updated the tag

**Review by TravisEz13 — CHANGES_REQUESTED** [review]

**Review by TravisEz13 — APPROVED** [review]

**Comment by TravisEz13** [comment]
@PoshChan Please remind me in 1 hour

**Comment by PoshChan** [comment]
@TravisEz13, this is the reminder you requested 1 hour ago

File: release/community-stable/oraclelinux/meta.json (hunk 1 above)
Comment (kiazhi) [reply release/community-stable/oraclelinux/meta.json:9]: Sorry but what's the conclusion about this?

Comment (RDIL) [reply release/community-stable/oraclelinux/meta.json:9]: It most likely would be better to do this with the shorttag if other linux images do it.

File: release/community-stable/oraclelinux/dependabot/Dockerfile (hunk 3 above)
Comment (kiazhi) [reply release/community-stable/oraclelinux/dependabot/Dockerfile]: do you mean that when I need to bump up the image version, I will have to add a dockerfile in the "/release/community-stable/oraclelinux/docker" folder?

PS: sorry, I'm learning this for the first time.

File: release/community-stable/oraclelinux/meta.json (hunk 1 above)
Comment (kiazhi) [reply release/community-stable/oraclelinux/meta.json:9]: I don't mind to accept the suggestion to conform the standard. But that would mean we need to fix the following below too.

https://github.com/PowerShell/PowerShell-Docker/blob/80312673876a39d3acdebf1ad202d6d29ad89bad/release/stable/centos7/meta.json#L6-L8

https://github.com/PowerShell/PowerShell-Docker/blob/80312673876a39d3acdebf1ad202d6d29ad89bad/release/stable/centos8/meta.json#L6-L8

https://github.com/PowerShell/PowerShell-Docker/blob/80312673876a39d3acdebf1ad202d6d29ad89bad/release/preview/centos7/meta.json#L6-L8

https://github.com/PowerShell/PowerShell-Docker/blob/80312673876a39d3acdebf1ad202d6d29ad89bad/release/preview/centos8/meta.json#L6-L8

**Review by TravisEz13 — APPROVED** [review]

**Pull request merged**